Xamarin.Forms.Platform.iOS/Renderers/RefreshViewRenderer.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.ComponentModel;
+using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	public class RefreshViewRenderer : ViewRenderer<RefreshView, UIView>
```

Comment:
Android registers an effect provider. Does iOS need to do the same?

Reply:
Yes, I just included it.